### DISCUSSION TITLE
Which process to kill??

### DISCUSSION THREAD

**Discussion by vulture2600** [David00/rpi-power-monitor#128] [General]
So when I try to run "sudo python3 power_monitor.py" it tells me that user root is running this in the back ground and cannot be run twice. It then lists a process number to kill but its the most current process (literally the command I just did) and if I try to kill it, it says no such process. I've tried "sudo ps -aux" but nothing running looks like the right one. What name would it be under?

Thanks!

**Comment by David00** [comment]
Can you run `sudo systemctl status power-monitor.service` and see if the service status is listed as active or not?  If so, this is the instance running in the background and you can stop it with `sudo systemctl stop power-monitor.service`.

The service should show up under `sudo ps -aux` so I wonder if perhaps this is an issue with how I implemented the 2nd process checking.

**Comment by vulture2600** [comment]
It occurred to me to try that about 5 minutes after I posted this.
power-monitor.service failed to start. I eventually figured out the config.toml file needs the very first line commented out, and all numerical decimal values need a leading zero.
.5 will not work, but 0.5 works fine. I got the service started and I'm able to run the console.
